apps/integration_tests/selenium_cases.py

```diff
@@ -325,6 +328,24 @@ class Action(Enum):
     },
 ]
 
+SEQ_AUTHORIZE_LANG_PARAM_START = [
+    {
+        "display": "Load BB2 Landing Page ...",
+        "action": Action.LOAD_PAGE,
+        "params": [HOSTNAME_URL]
+    },
+    CLICK_TESTCLIENT if not HOSTNAME_URL.startswith(PROD_URL) else LOAD_TESTCLIENT_HOME,

```

Comment:
In the deployment process, when do integration tests run? Looks like this test would get run either when running locally or when deploying to prod. Is that right? @sharonfruit

Reply:
just saw below that there are also integration tests for when running against an actual SlSx server like TEST

Reply:
The non Spanish Selenium tests are part of the continuous integration flow when we commit code. They're also part of the smoke tests that we manually run when we do a deploy which is why the scripts take the target system as a parameter.

Reply:
Thank you for explaining!